test/build/goldens/build-config-dev-env.golden.json

```diff
@@ -63,6 +63,7 @@
       "slider": "/demos/slider.scss",
       "snackbar": "/demos/snackbar.scss",
       "switch": "/demos/switch.scss",
+      "tab": "/demos/tab.scss",
```

Comment:
Ask Andy if we still need these golden files. I think the webpack refactor is over...so maybe we can remove these files

Reply:
Confirmed with @acdvorak these are necessary for now.